### ISSUE TITLE
Delete unused analytics files

### ISSUE THREAD

**Issue by david-venhoff** [digitalfabrik/integreat-cms#3043]
### Describe the Bug

I noticed that we have some unused files in the cms:
* `_broken_links_widget.html`
* `_translation_overview_widget.html`
* `_website_size_widget.html`
* `app_size.html` and `app_size_view.py`

These files appear to have been part of the original dashboard, before it was overhauled, but I am not completely sure about that.

I think we should delete these files, if they serve no purpose anymore.

**Comment by ulliholtgrave** [comment]
Good catch, most of the mentioned files are just placeholders for the old dashboard. However, `app_size.html `and `app_size_view.py` are connected to https://github.com/digitalfabrik/integreat-cms/issues/78. 
@osmers Maybe the service team should decide whether we still want this app size information or if this became obsolete?

**Comment by osmers** [comment]
Can we maybe keep the two files relating to the app size information for now? 
IMHO it is not a super relevant or urgent issue to show the app size in the CMS - one bcs even municipalities who knew the functionality never asked for it since the migration and two, most phones have more than enough space and Integreat only makes up a small portion of that...but in case we wanna integrate it in the future, we could keep the two files for now.
